packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss

```diff
@@ -180,6 +180,7 @@
 
   & button {
     width: 100%;
-    padding-inline-end: 0;
+    padding-inline: 15px 0;
```

Comment:
Please use the Carbon spacing name space for this (this is about `spacing.$spacing-05`; yes we need to clean this file up.

Reply:
done